Sources/SWBQNXPlatform/QNXLd.xcspec

```diff
@@ -153,18 +153,6 @@
                 );
                 IsInputDependency = Yes;
             },
-            {
-                Name = "LD_EXPORT_GLOBAL_SYMBOLS";
-                Type = Boolean;
-                DefaultValue = NO;
-                CommandLineArgs = {
-                    YES = (
-                        "-Xlinker",
-                        "-export_dynamic",
-                    );
-                    NO = ();
-                };
-            },
```

Comment:
I'm assuming `-rdynamic` will work here, but I don't have a way to verify.

Reply:
The qcc/q++ driver interface is notably different from clang or gcc even though it's based on gcc, and `-rdynamic` is not explicitly documented there, so sometimes it's hard to tell.

That said, I just tried `touch foo.c && qcc -v foo.c` and it appears to pass the right flags to the linker (which in the QNX toolchain is GNU ld).

https://www.qnx.com/developers/docs/8.0/com.qnx.doc.neutrino.utilities/topic/q/qcc.html

Reply:
Thanks!